src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -970,7 +970,10 @@ abstract class ClassfileParser {
           val s = module.info.decls.lookup(n)
           if (s != NoSymbol) Some(LiteralAnnotArg(Constant(s)))
           else {
-            warning(s"""While parsing annotations in ${in.file}, could not find $n in enum $module.\nThis is likely due to an implementation restriction: an annotation argument cannot refer to a member of the annotated class (scala/bug#7014).""")
+            warning(
+              sm"""While parsing annotations in ${in.file}, could not find $n in enum ${module.nameString}.
+                  |This is likely due to an implementation restriction: an annotation argument cannot refer to a member of the annotated class (scala/bug#7014).""".stripMargin
```

Comment:
I think you don't need to invoke `stripMargin` when using the `sm` interpolator

Reply:
Ach, you're right. I'm so used to that tailing triple-quoted strings that it's easy to ignore.

Is there a reason why `sm` couldn't grace the 2.13 stdlib?

Reply:
+1 Together with intrinsic standard interpolators.

Reply:
The latter is in a WIP PR. I'll pull over `sm` today.